src/scss/hpe/_hpe.defaults.scss

```diff
@@ -16,7 +16,7 @@ $brand-status-colors: (
 );
 $brand-link-color: inherit;
 $link-hover-color: #000;
-$brand-link-text-decoration: underline;
+$brand-link-text-decoration: none;
```

Comment:
I don't think this is the correct brand style. Where in BrandCentral is this causing issues?